tests/components/braviatv/test_config_flow.py

```diff
@@ -422,6 +423,43 @@ async def test_options_flow(hass):
         assert config_entry.options == {CONF_IGNORED_SOURCES: ["HDMI 1", "HDMI 2"]}
 
 
+async def test_options_flow_error(hass: HomeAssistant) -> None:
```

Comment:
Added a test for the case where `await coordinator.async_update_sources()` fails with `BraviaTVError`